### DISCUSSION TITLE
Do I install the package as a dev dependency?

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [joe-bell/cva#200] [Q&A] [answered]
Hello everyone, 

I have size-anxiety and I try to avoid installing any dependency in my UI kit library.

Do I install `class-variance-authority` as a dev dependency or do I have to install it as a regular dependency?

**Comment by bengry** [comment, marked as answer]
> Hello everyone,
> 
> I have size-anxiety and I try to avoid installing any dependency in my UI kit library.
> 
> Do I install `class-variance-authority` as a dev dependency or do I have to install it as a regular dependency?

1. It wouldn't _really_ matter. At the end of the day, given you use a bundler - if you `require`/`import` something and that's used by part of the bundled code, it's gonna get bundled - regardless of where it sits in the `package.json`.
2. a regular dependency (`dependencies`). Since the code is executed at runtime by the consumer. If you use SSR/SSG you could probably prevent this code from running on the client, but it would still run on the server at runtime/build time, respectively.

**Reply by TheMikeyRoss** [reply]
Awesome, thanks!
